src/app/api/util/email.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Should this folder be "util" singular or "utils" plural?
I noticed we have a different folder pluralized to "utils" below (L6) for an import from `"../../../utils/email"`.

Reply:
Fixed. Thanks